src/main/java/racingcar/ResultView.java

```diff
@@ -0,0 +1,38 @@
+package racingcar;
+
+import java.util.List;
+import java.util.Map;
+
+public class ResultView {
+
+    private Map<Integer, List<Integer>> record;
+    private int tryCount;
+
+    public ResultView(Map<Integer, List<Integer>> record) {
+        this.record = record;
+        tryCount = record.size();
+    }
+
+    public void show() {
+        System.out.println("실행결과");
+
+        for (int count = 1; count <= this.tryCount; count++) {
+            showCarPositions(record.get(count));
+            System.out.println();
+        }
+    }
+
+    private void showCarPositions(List<Integer> positions) {
+        for (Object position : positions) {
+            showGraph((Integer) position);
+        }
+    }
+
+    private void showGraph(int position) {
+        StringBuilder sb = new StringBuilder();
+        for (int j = 0; j < position; j++) {
+            sb.append("-");
+        }
+        System.out.println(sb.toString());
```

Comment:
```suggestion
    private void showCarPositions(List<Integer> positions) {
        positions.stream()
                .map(position -> CAR.repeat(position))
                .forEach(System.out::println);
    }
```
Stream API를 활용하여 반복문을 내부 라이브러리에 위임함으로써 무엇을 연산할지 적극적으로 드러내는 것은 어떨까요